packages/react/.storybook/preview.tsx

```diff
@@ -33,9 +33,12 @@ import {extendTheme} from '../src/theme';
  *
  * @param Story - Story component.
  * @param context - Story context.
- * @returns Stroy wrapped in providers.
+ * @returns Story wrapped in providers.
  */
 const withProviders = (Story: Story, context: StoryContext) => {
+  // Set 'oxygen-mode' to the Storybook theme
+  localStorage.setItem('oxygen-mode', JSON.parse(localStorage.getItem('sb-addon-themes-3'))?.current);
```

Comment:
Should this go in a default use effect hook?

Reply:
Fixed https://github.com/wso2/oxygen-ui/pull/147/commits/ea950367ad160ba7bd0b2bb815c7e1bcd2e5af09